<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::BookmarkPropertiesDialog</class>
 <widget class="QDialog" name="Otter::BookmarkPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionTextEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
       <property name="buddy">
        <cstring>addressLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Folder:</string>
       </property>
       <property name="buddy">
        <cstring>folderComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="folderLayout">
       <item>
        <widget class="Otter::BookmarksComboBoxWidget" name="folderComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newFolderButton">
         <property name="text">
          <string>Newâ€¦</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="Otter::LineEditWidget" name="addressLineEditWidget"/>
     </item>
     <item row="2" column="1">
      <widget class="Otter::LineEditWidget" name="titleLineEditWidget"/>
     </item>
     <item row="3" column="1">
      <widget class="Otter::TextEditWidget" name="descriptionTextEditWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="visitsLabel">
       <property name="text">
        <string>Visits:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lastVisitLabel">
       <property name="text">
        <string>Last visit:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="addedLabel">
       <property name="text">
        <string>Created:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Otter::TextLabelWidget" name="visitsLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Otter::DateTimeTextLabelWidget" name="lastVisitLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Otter::DateTimeTextLabelWidget" name="addedLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="modifiedLabel">
       <property name="text">
        <string>Modified:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Otter::DateTimeTextLabelWidget" name="modifiedLabelWidget" native="true">
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="keywordLabel">
       <property name="text">
        <string>Keyword:</string>
       </property>
       <property name="buddy">
        <cstring>keywordLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Otter::LineEditWidget" name="keywordLineEditWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::BookmarksComboBoxWidget</class>
   <extends>QComboBox</extends>
   <header>src/ui/BookmarksComboBoxWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::DateTimeTextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/DateTimeTextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextEditWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>src/ui/TextEditWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>folderComboBox</tabstop>
  <tabstop>newFolderButton</tabstop>
  <tabstop>addressLineEditWidget</tabstop>
  <tabstop>titleLineEditWidget</tabstop>
  <tabstop>descriptionTextEditWidget</tabstop>
  <tabstop>keywordLineEditWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
